drivers/led/lp5562.c

```diff
@@ -895,6 +903,26 @@ static inline int lp5562_led_off(const struct device *dev, uint32_t led)
 	return lp5562_led_set_brightness(dev, led, dev_data->min_brightness);
 }
 
+static int lp5562_led_update_current(const struct device *dev)
+{
+	const struct lp5562_config *config = dev->config;
+	int result = 0;
```

Comment:
Please use `ret`, just to be consistent with the other functions. There is also no need to initialize this variable to 0.